### PR TITLE
Use smaller ints for bitflags

### PR DESCRIPTION
**Pull request by Noratrieb**
Free shrinking!

### CODE REVIEW

File: compiler/rustc_driver/src/lib.rs [hunk 1]
```diff
@@ -2,3 +2,4 @@
 // `rustc_driver_impl` to be compiled in parallel with other crates.
 
 pub use rustc_driver_impl::*;
+//     meow
```
Comment (saethlin) [inline compiler/rustc_driver/src/lib.rs:5]:
```suggestion
// meow
```

Comment (compiler-errors) [reply compiler/rustc_driver/src/lib.rs:5]: what is this comment for lol?

Comment (jyn514) [reply compiler/rustc_driver/src/lib.rs:5]: 🐱

Comment (Nilstrieb) [reply compiler/rustc_driver/src/lib.rs:5]: i left that around when testing something else and accidentally added it

**Comment by compiler-errors** [comment]
r=me if perf comes back good (and comment is removed)

**Pull request merged**